src/Core/Language/LanguageManagerTrait.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace drunomics\ServiceUtils\Core\Language;
+
+use Drupal\Core\Language\LanguageManagerInterface;
+
+trait LanguageManagerTrait {
+    /**
+     * The config factory.
+     *
+     * @var \Drupal\Core\Language\LanguageManagerInterface
+     */
+    protected $languageManager;
+
+    /**
+     * Sets the config factory.
+     *
+     * @param \Drupal\Core\Language\LanguageManagerInterface $languageManager
+     *   The config factory.
+     *
+     * @return $this
+     */
+    public function setConfigFactory(LanguageManagerInterface $languageManager) {
+        $this->languageManager = $languageManager;
+        return $this;
+    }
+
+    /**
+     * Gets the config factory.
+     *
+     * @return \Drupal\Core\Language\LanguageManagerInterface
+     *   The config factory.
+     */
+    public function getConfigFactory() {
```

Comment:
getConfigFactory !? nonesense... Be sure to update everything as needed.
Take care! Also each trait must have a test added, can be done pretty much by search/replace also